src/FocusTrap.ts

```diff
@@ -54,13 +55,25 @@ export const FocusTrap = defineComponent({
 
   setup(props, { slots, emit }) {
     let trap: FocusTrapI | null
-    const el = ref<HTMLElement | null>(null)
+    const el = ref<HTMLElement | ComponentPublicInstance | null>(null)
 
     const ensureTrap = () => {
       if (trap) {
         return
       }
 
+      if (el === null || el?.value === null) {
+        throw new Error(
+          '[focus-trap-vue] FocusTrap requires exactly one child.'
+        )
+      }
+
+      if (!(el.value instanceof HTMLElement) && Reflect.has(el.value, '$el')) {
+        if (Reflect.get(el.value, '$el') instanceof HTMLElement) {
```

Comment:
Is there a reason to use Reflect on this?

Reply:
Was thinking it's better than doing `el.value?.$el`, but definitely open to your preference.

Reflect is, indeed, quite new and not necessarily a feature many would reach for, after all.

Reply:
`.?` should not be used in order to support older browsers

Reply:
If support for older browsers is a desired thing, then I guess `Object.hasOwnProperty` it is.

Reply:
Usually just using something like `el && el.property` to replace `el.?property` is fine 😄